src/components/test/Details.test.js

```diff
@@ -48,6 +48,7 @@ describe('Details component', () => {
     render(
       <Router>
         <Details />
+
       </Router>,
     );
 
```

Comment:
- [ ] Hi, I would advise you to work on the test again. Can you check the test locally before asking for a review, it will help you to save time. Here is the test result:
- 
![Capture](https://github.com/ANTOINE1128/React-capstone-project---Metrics-webapp/assets/63972670/c86118d6-fd38-4266-a493-500a52e028e1)